tests/integration/submissions/test_submissions_views.py

```diff
@@ -76,13 +74,41 @@ def test_new_author_submit_with_required_fields(
                 }
             }
         ),
-        headers=headers,
     )
     assert response.status_code == 200
 
+    mock_requests_post.assert_called_once()
```

Comment:
This is not actually how you should mock a request :)
Rather than mocking the request library, you should 'intercept' the request and return a fake response.
Give a look to https://requests-mock.readthedocs.io/en/latest/mocker.html

Reply:
don't we want to use something like `vcrpy` instead?

Reply:
I think is a bit overkill to use `vcr` just to reply a json with and ID

Reply:
The point is we will use `vcr` eventually for other stuff (orcid for example), so instead of doing the same thing in a different way, we might as well use it here too.